tests/test_detector.py

```diff
@@ -84,7 +86,7 @@ def duplicateAmpName(dw):
             DetectorWrapper(modFunc=duplicateAmpName)
 
         # These break in the pybind layer
-        for crosstalk in ([1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0],  # 1D and not numpy
+        for crosstalk in ([11.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0],  # 1D and not numpy
```

Comment:
Was this change intentional?

Reply:
No; used in setting a breakpoint when trying to find the numpy problem.  I'll revert it.